tests/foreman/ui/test_bookmarks.py

```diff
@@ -1,25 +1,79 @@
 """Test classes for Bookmark tests"""
 # -*- encoding: utf-8 -*-
-from robottelo.constants import BOOKMARK_ENTITIES
-from robottelo.decorators import stubbed, tier1, tier2
+from fauxfactory import gen_string
+from nailgun import entities
+from random import choice
+from robottelo.constants import BOOKMARK_ENTITIES, STRING_TYPES
+from robottelo.decorators import (
+    bz_bug_is_open,
+    skip_if_bug_open,
+    stubbed,
+    tier1,
+    tier2,
+)
 from robottelo.test import UITestCase
+from robottelo.ui.locators import common_locators, locators
+from robottelo.ui.session import Session
 
 
-def create_entity(entity):
-    """Take entity name and call nailgun methods to create a random instance
-
-    :param entity: Nailgun entity class to be processed
+def gen_random_string(length=10):
```

Comment:
That was my initial thought. Was thinking like that:
1) I can re-use datafactory's method and take random element
2) Well, i need to reuse that very often in each test, calling each time `random.choice(generate_strings_list())` is a bit ugly, better make some `gen_random_string()` helper.
3) Ok, as it's separate helper, it's time to optimize it - no need to generate a list of strings each time, i can create a random string and that's it, better use `gen_string()` with random choice of `STRING_TYPES`.